docs/getting-started.md

```diff
@@ -47,13 +47,13 @@ cd <projectName>
 <!--Using Yarn (Recommended)-->
 
 ```bat
-yarn add react-native-windows@canary
+yarn add react-native-windows@latest

```

Comment:
This main page in the docs needs to stay the same as it reflects how to build against main/canary/the perpetual next version.

```suggestion
yarn add react-native-windows@canary

```